client/asset/btc/spv.go

```diff
@@ -1196,6 +1197,81 @@ func (w *spvWallet) startWallet() error {
 	return nil
 }
 
+func (w *spvWallet) rescanWalletAsync() error {
```

Comment:
This approach is all to keep the neutrino chain service running and only touch the wallet and its DB.  Otherwise we could have rescan handled with a full stop / drop / start, mostly coordinated higher up by the ExchangeWallet.
I'm still not certain this approach is flawless, but the wallet does go back to running as normal, eventually anyway.